packages/terra-notification-dialog/translations/es-US.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Terra.notification.dialog.info": "Information.",
```

Comment:
Is the `.` after Information a typo?

Reply:
Looks like it. Changed it and propagated the change to other translation files.